protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3654,9 +3658,13 @@ message ReferralSetRefereeConnection {
 // Request to retrieve information about a referral set's referees
 message ListReferralSetRefereesRequest {
   // Referral set ID to retrieve information for
-  string referral_set_id = 1;
+  optional string referral_set_id = 1;
   // Pagination information to limit the data that is returned.
   optional Pagination pagination = 2;
+  // Referrer to filter by. If referrer set id is provided, this field is ignored.
```

Comment:
according to the docs guidelines we should be using `ID` not `id`

https://github.com/vegaprotocol/vega/blob/develop/DOCUMENTATION_STYLE.md#what-not-to-use-and-what-to-use-instead